package.json

```diff
@@ -2,10 +2,12 @@
   "name": "1on1-questions",
```

Comment:
this is the name under the npm package will be published, in case you want to change it